src/sadhana/MeetingRoom.java

```diff
@@ -0,0 +1,41 @@
+package sadhana;
+
+/* Assignment_40
+ Program - 2:  
+Create a class MeetingRoom having method bookMeetingRoom which should provide
+below facility.
+meeting room : Nalanda, Takshashila, AgraFort, PratapGadh
+a. �Nalanda� meeting room will be booked with default bookMeetingRoom() method
+for 1 hr.
+b. Write another method which helps to book Other then �Nalanda� meeting room by
+giving name of meeting room and hrs as parameter.
+c. displayInfo method should print booked meeting room info [i.e. meeting room,
+booked for how many hrs] */
+
+public class MeetingRoom {
+
+	String meetingRoom;
+	int timeInhrs;
+
+	void bookMeetingRoom() {
+		this.meetingRoom = "Nalanda";
+		this.timeInhrs = 1;
+	}
+
+	void otherMeetingRoom(String meetingRoom, int timeInhrs) {
+		this.meetingRoom = meetingRoom;
```

Comment:
you should put condition that if Room is nalanda then call bookingRoom() method else book based on requirement.